pandas/tests/scalar/period/test_period.py

```diff
@@ -1549,9 +1550,8 @@ def test_period_immutable():
 
 
 @pytest.mark.xfail(
-    # xpassing on MacPython with strict=False
-    # https://travis-ci.org/MacPython/pandas-wheels/jobs/574706922
-    PY35,
+    # Bug in old versions of python-dateutil (failing in 2.6.1)
+    StrictVersion(dateutil.__version__) <= StrictVersion("2.6.1"),
```

Comment:
Failing because of dateutil, not Python 3.5

Reply:
Looks like dateutil 2.7.0 was released in March 2018.  I think we're safe to bump up to at least that.

Either way, really glad to see the reason for the failing test identified.

Reply:
[Here is the version breakdown](https://libraries.io/pypi/python-dateutil/usage) of applications pinning `dateutil` versions. I think a disproportionate number are pinning `2.4.*` because of the way fallback options for ambiguous YMD/MDY/DMY triples changed after that version.

I think it's probably fine to pin to a more recent version, but I'm not *entirely* sure it's warranted based on the fact that some people might experience a specific bug if they are using an old version of dateutil. I'd probably just leave the skip condition in place on the test until you have some actual new feature of `dateutil` that you want to use, but in the long run I'm not clear how much it will really matter.